dist/index.js

```diff
@@ -74457,6 +74457,11 @@ const useYarn = () => fs.existsSync(yarnFilename)
 const usePnpm = () => fs.existsSync(pnpmLockFilename)
 const useNpm = () => fs.existsSync(packageLockFilename)
 
+let runPrefix = 'npx'
+if (useYarn()) {
+  runPrefix = 'yarn'
+}
```

Comment:
```suggestion
const runPrefix = useYarn() ? 'yarn' : 'npx'
```

How about this to use a `const` versus `let` variable?

Reply:
@warrensplayer 

You have commented here on the compiled code, so effectively it's a duplicate of your other comment.

The change you requested has been implemented by editing the source code `index.js` and then by executing `npm run build` which updates `dist/index.js`,